### PR TITLE
Bash script for seeding test data, fixed test data : Caleb

### LINKED ISSUE #155: Create seed test data shell script
As a developer I want to use shell scripts to connect to the project RDS and populate it with test data when working on code so that this process is more streamlined and does not require as much time.

- Create `seed_test.sh`
- Have it populate RDS with fake data

### PR DESCRIPTION
**Pull request by CalebMathers**
This closes #155 

Wrote a shell script that just runs the fake data sql inserts on the RDS database.
Also fixed the fake_data.sql to account for the addition of the new column in the body assignment table for constellation id.

### CODE REVIEW

File: database/fake_data.sql [hunk 1]
```diff
@@ -218,30 +218,30 @@ VALUES
...
+(3, 3, '2024-10-01 15:45:00', 140.3, 20.8, 4),
+(4, 4, '2024-10-01 16:00:00', 185.0, 25.7, 5),
+(5, 5, '2024-10-01 16:15:00', 230.6, 35.8, 6),
+(6, 6, '2024-10-01 16:30:00', 275.4, 45.3, 7),
+(7, 7, '2024-10-01 16:45:00', 320.2, 15.4, 8),
+(8, 8, '2024-10-01 17:00:00', 365.1, 50.9, 9),
+(9, 9, '2024-10-01 17:15:00', 30.7, 60.8, 10),
+(10, 1, '2024-10-01 17:30:00', 80.4, 55.5, 1),
+(11, 3, '2024-10-01 17:45:00', 120.9, 65.7, 4),
+(12, 4, '2024-10-01 18:00:00', 150.2, 70.3, 5);
```
Comment (AlexOBarnes) [inline database/fake_data.sql:247]: I love how we have used fake data here

File: database/seed_test.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+source .env
+export PGPASSWORD=$DB_PASSWORD
+psql -h $DB_HOST -p $DB_PORT -U $DB_USER $DB_NAME -f fake_data.sql
```
Comment (AlexOBarnes) [inline database/seed_test.sh:3]: Very helpful bash script

**Review by MurasatoNaoya — APPROVED** [review]
Fixed test data to account for recent schema / ERD changes to do with the _body_ table. LGTM.

**Review by AlexOBarnes — APPROVED** [review]
Thank you for streamlining this process

**Pull request merged**